nntrainer/tensor/manager.cpp

```diff
@@ -387,6 +390,10 @@ std::vector<Weight *> Manager::requestWeights(
                                         var_ls, t_initializer);
 
       if (trainable && need_gradient) {
+        /** We cannot use the tensor shedulding for weight gradient if the
```

Comment:
It's trivial, but there's a typo error in the spelling of scheduling.

Reply:
fixed. Thanks.